src/cpu/pred/bpred_unit.cc

```diff
@@ -172,37 +170,30 @@ BPredUnit::predict(const StaticInstPtr &inst, const InstSeqNum &seqNum,
         //       support coroutines.
```

Comment:
Ohhhh. Is it this note?
To support coroutines I should allow both, push and pop, in one instruction?